topics/dotnet/index.md

```diff
@@ -5,9 +5,9 @@ display_name: ".NET"
 github_url: https://github.com/dotnet
 logo: dotnet.png
 released: February 13, 2002
-short_description: ".NET is a software framework developed by Microsoft."
+short_description: ".NET is a free, cross-platform, open source developer platform."
 topic: dotnet
-url: https://www.microsoft.com/net/
+url: https://dotnet.microsoft.com
 wikipedia_url: https://en.wikipedia.org/wiki/.NET_Framework
 ---
-.NET is a software framework developed by Microsoft in February of 2002 as an infrastructure for building and running applications and services. The purpose of .NET is to make it easier for users to share information between websites. The framework is intended to be utilized for Windows platforms.
+.NET is a free, cross-platform, open source developer platform for building many different types of applications.
```

Comment:
Same as comment above. Need to distinguish between .NET Core and .NET Framework

Reply:
That matches the description on https://dotnet.microsoft.com so, yay for that :-) 👍